doc/pages/changelog.asciidoc

```diff
@@ -5,9 +5,8 @@ released versions.
 
 == Development version
 
-* History is now stored linearly instead of in a tree
-
-* `<a-u>` and `<a-U>` now undo selection history
+* Selection undo/redo moved to `<a-u>`/`<a-U>` and moving in history tree to
```

Comment:
"moved" is wrong because it hadn't been released yet

```suggestion
`<a-u>` and `<a-U>` now undo selection changes; the previous meaning of moving in history tree has been oved to `<c-j>`/`<c-k>`.
```

Same in main.cc

Reply:
Agreed.